src/app/components/result-box/result-box.component.ts

```diff
@@ -1,17 +1,19 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-result-box',
   templateUrl: './result-box.component.html',
   styleUrls: ['./result-box.component.scss'],
 })
 export class ResultBoxComponent {
-  @Input() word = '';
-  @Input() borderResultStyles = false;
+  @Input() word = ''
+  @Input() splitWord: String[] = [];
+  @Input() borderResultStyles!: boolean;
 
   borderStyles(): String{
+    if (this.borderResultStyles === undefined) return '';
```

Comment:
```suggestion
    if (this.borderResultStyles && firstRound) return '';
```

Pass another flag from parent telling the component that should not paint borders since there's no guessing word yet